src/assets/text/droughtNarrations_desktop.js

```diff
@@ -107,7 +107,7 @@ export default {
         title: 'Turn-of-the-Century (1999 - 2015)',
         start_date: '1999-09-01',
         end_date: '2009-09-01',
-        text: 'This was one of the worst droughts to hit the southwestern U.S. in the last 1,200 years. Like previous droughts, low amounts of rain led to big drops in available water, but high temperatures drove the loss of soil moisture during this most recent drought (<a href="https://doi.org/10.1002/joc.7904" target="_blank">McCabe et al. 2022</a>). ',
+        text: 'This was one of the worst drought periods to hit the southwestern U.S. in the last 1,200 years. Like previous droughts, low amounts of rain led to big drops in available water, but high temperatures drove the loss of soil moisture during this most recent drought (<a href="https://doi.org/10.1002/joc.7904" target="_blank">McCabe et al. 2022</a>). ',
```

Comment:
Just a note - here you switch to using 'drought period' instead of just 'drought', but you use just 'drought' in the descriptions for the 1980s drought, 1960s drought, 1950s drought, and the dust bowl.